galsim/hsm.py

```diff
@@ -441,14 +440,14 @@ def EstimateShear(gal_image, PSF_image, weight=None, badpix=None, sky_var=0.0,
     @returns a ShapeData object containing the results of shape measurement.
     """
     # prepare inputs to C++ routines: ImageView for galaxy, PSF, and weight map
```

Comment:
Some outdated comments in this file (`_convertImage` doesn't return an ImageView after the changes in this PR).  I've flagged this one but I see more elsewhere.